### ISSUE TITLE
Incompatible with 2022.2.1

### ISSUE THREAD

**Issue by DmitriyZaitsev** [JetBrains/spek#996]
Hi @raniejade 
I get the message `Incompatible: requires IDE build 213.* or earlier` on the Android Studio Electric Eel | 2022.2.1 Canary 5 _#AI-221.4165.146.2211.8713749_, built on June 17, 2022.

Yet, I have to stay on Android Studio Electric Eel | 2022.1.1 Canary 3 _#AI-213.7172.25.2211.8624637_, built on May 21, 2022.

When will you have a chance to support the new version? Thanks!

**Comment by raniejade** [comment]
Thanks for the report @DmitriyZaitsev! I don't usually release builds for canary as they tend to move in build versions quite often, making it hard to keep up.

**Comment by DmitriyZaitsev** [comment]
@raniejade that's true about canaries, you're right! Wherein, I'm not reporting the issue related to the canary build. The plugin does not support 2022.2.1 at all. Moreover, I tried to install the plugin for the official IDEA CE 2022.1.2, but it doesn't work either.

Summing up, I kindly ask for support for 2022.1.2 and 2022.2.1.
Apologies for not being clear in my initial comment.

I love this project and I've been using it for five years maybe. Thank you for making this possible.
I'm probably using some wrong plugins, but they all somehow don't suffer from such compatibility issues, they all just continue to work when the IDE is updated, but not the Spek. I would really like the spek to behave the same way.

I understand you don't have time for maintaining this repo. I'd love to help in any way I can, but I don't have experience developing IDE plugins. I could make a relevant pull request if you tell me where to look and where to dig. I would be interested in figuring this out.

Thanks!

**Comment by raniejade** [comment]
@DmitriyZaitsev ohh right. There should be a build in the dev channel if you would like to try: https://plugins.jetbrains.com/plugin/10915-spek-framework/versions/dev. I haven't done a release yet which should include support for 2022.* (I'll see if I have time this coming weekend).

**Comment by DmitriyZaitsev** [comment]
Hi @raniejade 
have you had a chance to look at this?

**Comment by raniejade** [comment]
Hey @DmitriyZaitsev not yet unfortunately - have you tried the dev build of the plugin? (You have to download it and install it manually)

**Comment by DmitriyZaitsev** [comment]
@raniejade I have.
For IDEA CE it at least installs.
For AS I get the following message:
```
Plugin 'Spek Framework' (version '2.0.19-alpha.0.2+04a479e-SNAPSHOT-Studio4.2') is not compatible with the current version of the IDE, because it requires build 202.7660.26.* or older but the current build is AI-221.5591.52
```

**Comment by koenpunt** [comment]
The moment is here:

<img width="681" alt="image" src="https://user-images.githubusercontent.com/351038/181058760-35351f8c-eaa5-4d5c-b3cb-a4041ea62188.png">

**Comment by jurriaan** [comment]
Since https://plugins.jetbrains.com/plugin/10915-spek-framework/versions/dev/176742 works fine with 2022.2, is it possible to officially release the latest version of the plugin?

**Comment by asubb** [comment]
Works for me with the latest Intellij IDEA 2022.2.2:
![image](https://user-images.githubusercontent.com/2813479/192904103-32be23d6-5140-4707-8db4-10acc77fad36.png)

**Comment by sraiteri** [comment]
Hey @raniejade - upon upgrading to Android Studio Eletric Eel (2022.1.1) released last week, we're noticing that the latest Spek Framework plugin version isn't appearing in the plugin Marketplace and still marked incompatible.

<img width="1334" alt="Screen Shot 2023-01-16 at 12 15 56 pm" src="https://user-images.githubusercontent.com/7417780/212585370-fd06c42b-43b5-4085-b1b5-4dc8c99eab95.png">

Any plans (or PR assistance required) to support AS 2022.1.1?

**Comment by DmitriyZaitsev** [comment]
I held back to the last, but, unfortunately, after five years of using this framework as the main and personally beloved, the time has come to abandon it in favor of Kotest due to the lack of tooling support.
I wish Spek prosperity and Ranie to find a permanent maintainer interested in this awesome project!

**Comment by Stampede10343** [comment]
Is this Repo dead? I'm also facing issues with not being able to use the Spek plugin on the latest version of Android Studio

**Comment by lnhrdt** [comment]
Hey @hhariri I just remembered a time on an episode of the Talking Kotlin podcast that you mentioned starting Spek. Just to check my memory, I peeked at the [contributors graph](https://github.com/spekframework/spek/graphs/contributors) and it confirmed as much! I know you haven't been part of the project for a while now but as a godfather of sorts for the project I thought I'd try to see if you could weigh in.

For the past 2 years The Spek Framework IntelliJ plugin has been slow to keep pace with IntelliJ updates. [According to](https://github.com/spekframework/spek/issues/989#issuecomment-1066068092) @raniejade the plugin needs some manual intervention by a maintainer to be compatible with new IntelliJ versions as they come out but there haven't been any maintainers able to do this for some time. I can tell from the several GitHub issues that many people still like Spek but are having problems with tooling support.

I feel it would be tragic if the project withered away primarily because the IntelliJ plugin needs minimal compatibility updates. Is there any chance you could help get things back on track? Or put out a call for a new maintainer? Or should the project just be archived and some migration recommendations be made for users, as @DmitriyZaitsev has done?
